doc/manuals/admin/database_model.md

```diff
@@ -194,9 +194,10 @@ Fields:
     we ensure that registration IDs are unique and that queries by
     registration IDs will be very fast (as there is an automatic default
     index in \_id).
--   **format**: the format to use to send forwarded requests. The only accepted value for now
-    is **JSON** (meaning NGSIv1 format), although this
-    may change in the future (see [issue about NGSIv2-based forwarding](https://github.com/telefonicaid/fiware-orion/issues/3068)).
+-   **format**: the format to use to send forwarded requests.
+    For NGSIv1 format, use **JSON** as value for `format`.
+    For NGSIv2, as of today, only **normalized** format is supported.
+    In the near future **keyValues** and **values** will be implemented as well (only for NGSIv2 forwarding).
```

Comment:
It's not sure (see https://github.com/telefonicaid/fiware-orion/pull/3467#discussion_r270308979). So, better not include this sentence in the documentation.

Reply:
Only thinking of Orion as context provider, it makes sense what you say, to not implement keyValues/values.  But, perhaps somebody wants to implement a context provider of their own and would like this feature ...
I wouldn't rule it out. The idea is good, in my opinion.
"Near future" that's perhaps a little bit optimistic.

That said, I removed the line in:
Fixed in 2ca5e66